### PR TITLE
Release 10.0.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,14 +4,30 @@ All notable changes to this project will be documented in this file.
...
+
+- Drop support for Debian 9, Ubuntu 16.04 and 18.04, FreeBSD 11, add support for Puppet 8, add support for RHEL 9 [\#453](https://github.com/voxpupuli/puppet-nodejs/pull/453) ([bastelfreak](https://github.com/bastelfreak))
+
+## [v9.1.0](https://github.com/voxpupuli/puppet-nodejs/tree/v9.1.0) (2023-09-16)
 
 [Full Changelog](https://github.com/voxpupuli/puppet-nodejs/compare/v9.0.1...v9.1.0)
 
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#467](https://github.com/voxpupuli/puppet-nodejs/pull/467) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (evgeni) [inline CHANGELOG.md:21]: Should we yank the 9.1.0 release because of that?

**Comment by evgeni** [comment]
Could also wait for https://github.com/voxpupuli/puppet-nodejs/pull/480 :)

File: CHANGELOG.md (hunk 1 above)
Comment (zilchms) [reply CHANGELOG.md:21]: in my opinion this is basically a breaking change und should require a major release. but maybe there are some other opinions (puppet6 is EOL so...)

**Comment by zilchms** [comment]
> Could also wait for #480 :)

Seems like a good idea, shouldnt take too long to get that one merged

**Comment by evgeni** [comment]
> > Could also wait for #480 :)
> 
> Seems like a good idea, shouldnt take too long to get that one merged

So I thought, but then we don't have Debian 12 support in puppet_metadata yet and Ubuntu 22.04 tests seem to timeout for some funny reason I couldn't care about right now.

File: CHANGELOG.md (hunk 1 above)
Comment (evgeni) [reply CHANGELOG.md:21]: That's also my reasoning here.

Ideally we would yank that release from Forge and tell GHC to fold the changelog into the 10.0 one, but I have no idea how.

Comment (evgeni) [reply CHANGELOG.md:21]: GHC has a `exclude-tags` option, but I don't seem to be able to get it to work :cry:

Comment (zilchms) [reply CHANGELOG.md:21]: on the other hand: whats done is done, yanking it now after the fact may create even more problems for others. i vote for just leaving it the way it is now

**Comment by zilchms** [comment]
then lets leave it out for now, we could throw that in a 10.1.0 release if necessary

**Review by zilchms — APPROVED** [review]

**Pull request merged**